### DISCUSSION TITLE
File-Suffixes duplicated in Downloaded Album

### DISCUSSION THREAD

**Discussion by fmaack** [LycheeOrg/Lychee#2440] [Bugs] [answered]
### Lychee version

v5.3

### Which PHP version are you using?

PHP 8.3

### Detailed description of the problem

I am using the "Download Album"-Feature which creates a zip-file containing all the images.

The filenames of the images inside the zip-file contains the filetype two times.

### Steps to reproduce the issue

1. upload an image IMG_123.jpeg into an album.
2. use the "Download Album" Button.
3. unpack the zip-File.

Result: The downloaded filename ist IMG_123.jpeg.jpeg.

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by ildyria** [comment, marked as answer]
Fixed.
